enterprise/backend/test/metabase_enterprise/sandbox/api/field_test.clj

```diff
@@ -22,67 +22,82 @@
 
 (deftest field-values-test
   (testing "GET /api/field/:id/values"
-    (mt/with-gtaps {:gtaps      {:venues {:query      (mt.tu/restricted-column-query (mt/id))
-                                          :remappings {:cat [:dimension (mt/id :venues :category_id)]}}}
-                    :attributes {:cat 50}}
-      (testing (str "When I call the FieldValues API endpoint for a Field that I have segmented table access only "
-                    "for, will I get ad-hoc values?\n")
-        (letfn [(fetch-values [user field]
-                  (-> (mt/user-http-request user :get 200 (format "field/%d/values" (mt/id :venues field)))
-                      (update :values (partial take 3))))]
-          ;; Rasta Toucan is only allowed to see Venues that are in the "Mexican" category [category_id = 50]. So
-          ;; fetching FieldValues for `venue.name` should do an ad-hoc fetch and only return the names of venues in
-          ;; that category.
-          (is (= {:field_id        (mt/id :venues :name)
-                  :values          [["Garaje"]
-                                    ["Gordo Taqueria"]
-                                    ["La Tortilla"]]
-                  :has_more_values false}
-                 (fetch-values :rasta :name)))
+    (mt/with-temp-copy-of-db
```

Comment:
the test itself was not updated, what changed in this test is I added another round of test where GTAP use a native question with parameter.